command/node_status.go

```diff
@@ -103,6 +103,11 @@ func (c *NodeStatusCommand) Run(args []string) int {
 		return 1
 	}
 
+        var attributes []string
+        for k, v := range node.Attributes {
+                attributes = append(attributes, fmt.Sprintf("%s:%s", k, v))
+        }
```

Comment:
I would sort the keys and then create the attributes by iterating through the sorted keys and making the list. That way looking for a particular attribute in the output will be a lot easier!